argo-cd-apps/base/toolchain-member/kustomization.yaml

```diff
@@ -3,4 +3,4 @@ kind: Kustomization
 resources:
   - toolchain-member-operator.yaml
 components:
-  - ../../../../k-components/deploy-to-member-cluster-merge-generator
\ No newline at end of file
+  - ../../k-components/inject-infra-deployments-repo-details
```

Comment:
is this not supposed to be `../../k-components/assign-toolchain-member-role-to-local-cluster`

Reply:
This is intentional (see my other comment about it). That being said, I'm not sure if we need this `inject-infra-deployments-repo-details` k-component. It seems to be applied to some appsets but not others.